docarray/array/storage/elastic/backend.py

```diff
@@ -34,6 +34,8 @@ class ElasticConfig:
     index_name: Optional[str] = None
     es_config: Dict[str, Any] = field(default_factory=dict)
     batch_size: int = 64
+    ef_construction: Optional[int] = None
```

Comment:
why don't we put directly the defaults here?

Reply:
please follow up on this thread:
https://jinaai.slack.com/archives/C02AC4T8Y5T/p1649754663487439
Me and the team are with option 1 for the mentioned arguments, lmk your opinion